admin-front/src/components/Dashboard/Dashboard.tsx

```diff
@@ -137,20 +147,27 @@ export const Dashboard = () => {
     <div>
       <div style={styles.flexColumn as React.CSSProperties}>
         <Welcome />
-        <CardWithIcon
-          to="/"
-          icon={EuroIcon}
-          title="Total revenue"
-          subtitle={`${totalNFTPriceRevenue} EUR`}
-        />
-        <VerticalSpacer />
-        <CardWithIcon
-          to="/"
-          icon={ShoppingCartRoundedIcon}
-          title="Nb of sold nfts (24h)"
-          subtitle={totalNFTCount24h}
-        />
-        <VerticalSpacer />
+        {
+          permissions.indexOf(1) !== -1 ? (
```

Comment:
I don't think roles should be hardcoded like this. Can't FE call an endpoint of BE to get all roles and thereafter use this info to map from name to identifier?